services/movie_session.py

```diff
@@ -0,0 +1,39 @@
+from db.models import MovieSession
+from datetime import datetime
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
+    return MovieSession.objects.create(show_time=movie_show_time,
+                                       movie_id=movie_id,
+                                       cinema_hall_id=cinema_hall_id)
+
+
+def get_movies_sessions(session_date: datetime = None) -> MovieSession:
+    if session_date:
+        return MovieSession.objects.get(show_time__date=session_date)

```

Comment:
The `get_movies_sessions` function uses `get` to retrieve sessions by date, which will raise an error if there is more than one session on the given date. Consider using `filter` instead to return a queryset of all sessions on that date.